graphs/possible_bipartition.py

```diff
@@ -5,8 +5,91 @@ def possible_bipartition(dislikes):
     """ Will return True or False if the given graph
         can be bipartitioned without neighboring nodes put
         into the same partition.
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: O(V+E) v = vertices, e = edges
+        Space Complexity: O(V+E)     
```

Comment:
✨ Very nice, however I believe space complexity would be O(V) here where V is the number of dogs/nodes in the graph as the data structures you create only ever hold, at most, all the dogs/nodes in the graph (not all of the edges as well).